client/src/explorer/model/AdminIdentityTreeItem.ts

```diff
@@ -19,8 +19,16 @@ import { BlockchainExplorerProvider } from '../BlockchainExplorerProvider';
 export class AdminIdentityTreeItem extends BlockchainTreeItem {
     contextValue: string = 'blockchain-admin-identity-item';
 
-    constructor(provider: BlockchainExplorerProvider, public readonly label: string, public readonly walletName: string) {
+    constructor(provider: BlockchainExplorerProvider, public readonly label: string, public readonly walletName: string, public readonly attributes: any = {}) {
```

Comment:
if you make this extend an identity tree item you won't need the same code twice